spec/controllers/api/v1/jobs/update_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+describe API::V1::Jobs::Update, type: :request do
+  let(:endpoint) { "/api/v1/jobs/#{id}" }
+  let!(:job) { create(:job) }
+  let(:id) { job.id }
+
+  subject { patch endpoint, params: params }
+
+  let(:params) { attributes_for(:job, title: title) }
+
+  context 'with valid params' do
+    let(:title) { 'Meticulous code reviewer' }
+    it 'should return 200' do
+      subject
+      expect(response).to have_http_status :ok
+    end
+
+    it 'should have new title' do
+      subject
+      expect(job.reload.title).to eq(params[:title])
```

Comment:
Since you use `let(:title)` I think you should be able to check if `job.title` is equal to this created `title`